tests/test_edgeql_insert.py

```diff
@@ -3275,6 +3275,16 @@ async def test_edgeql_insert_unless_conflict_25(self):
             q, [], variables={'n': "2"},
         )
 
+    async def test_edgeql_insert_unless_conflict_26(self):
+        # Test unless conflict on a property not actually mentioned
```

Comment:
Would be great to add a variant of this test where we _actually_ get a conflict, e.g when a conflicting value is from a default.

Reply:
We have tests for the cases with defaults. This is for when there's *not* a default